main.cpp

```diff
@@ -10,158 +9,266 @@ video: Chapter 2 - Part 3
+{ 
+    ignoreUnused(number); 
+    return 0; // Placeholder implementation, actual implementation required
+}  
+
+// 9) Function to convert miles per hour to kilometers per hour 
+float mphToKph(float mph) 
+{ 
+    ignoreUnused(mph); 
+    return mph * 1.60934f; 
+}  
+
+// 10) Function to find the minimum of two numbers 
+float findMinimum(float num1, float num2) 
+{ 
+    ignoreUnused(num1, num2); 
+    return std::min(num1, num2); 
+}
+
 
-/*
- 1)
- */
-
-/*
- 2)
- */
-
-/*
- 3)
- */
-
-/*
- 4)
- */
-
-/*
- 5)
- */
-
-/*
- 6)
- */
-
-/*
- 7)
- */
-
-/*
- 8)
- */
-
-/*
- 9)
- */
-
-/*
- 10)
- */
 
 /*
  MAKE SURE YOU ARE NOT ON THE MASTER BRANCH
 
- Commit your changes by clicking on the Source Control panel on the left, entering a message, and click [Commit and push].
- 
- If you didn't already: 
+ Commit your changes by clicking on the Source Control panel on the left,
+ entering a message, and click [Commit and push].
+
+ If you didn't already:
     Make a pull request after you make your first commit
-    pin the pull request link and this repl.it link to our DM thread in a single message.
+    pin the pull request link and this repl.it link to our DM thread in a single
+ message.
 
  send me a DM to review your pull request when the project is ready for review.
 
  Wait for my code review.
  */
 
-int main()
-{
-    //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
-    auto carRented = rentACar(6, 2); 
-    
-    //1)
-    
-    //2)
-    
-    //3)
-    
-    //4)
-    
-    //5)
-    
-    //6)
-    
-    //7)
-    
-    //8)
-    
-    //9)
-    
-    //10)
-    
-    
-    ignoreUnused(carRented);
-    std::cout << "good to go!" << std::endl;
-    return 0;    
+int main() {
+  // example of calling that function, storing the value, and passing it to
+  // ignoreUnused at the end of main()
+  auto carRented = rentACar(6, 2);
+
+    // 1)
+    auto rectangleArea = calculateRectangleArea(4.2, 7.8);
+
+    // 2)
+    auto boxVolume = calculateBoxVolume(3.5, 2.0, 6.3);
+
+    // 3)
+    auto celsiusValue = fahrenheitToCelsius(89.6);
+
+    // 4)
+    auto fahrenheitValue = celsiusToFahrenheit(37.0);
+
+    // 5)
+    auto isNumberEven = isEven(15);
+
+    // 6)
+    auto isNumberPrime = isPrime(23);
+
+    // 7)
+    auto maximumNumber = findMaximum(10, 25, 18);
+
+    // 8)
+    auto factorialValue = calculateFactorial(5);
+
+    // 9)
+    auto kphSpeed = mphToKph(65.0);
+
+    // 10)
+    auto minimumNumber = findMinimum(3.7f, 8.9f);
+
+    
+
+  
+  ignoreUnused(carRented);
+    ignoreUnused(rectangleArea);
+    ignoreUnused(boxVolume);
+    ignoreUnused(celsiusValue);
+    ignoreUnused(fahrenheitValue);
+    ignoreUnused(isNumberEven);
+    ignoreUnused(isNumberPrime);
+    ignoreUnused(maximumNumber);
+    ignoreUnused(factorialValue);
+    ignoreUnused(kphSpeed);
+    ignoreUnused(minimumNumber);
+    
+  std::cout << "good to go!" << std::endl;
+  return 0;
 }

```

Comment:
```
9) in the main function at the end: 
    for each of those functions declared, 
        c) pass the local variables to ignoreUnused() as you did in 2b)
```
- [ ] revise
- [ ] Fix the indentation so that the code conforms to the course coding standard